NSS-Spontaneous-Spoonbills/Sprint2
# Js form employees

## Link to Ticket

Autolink format: hashtag and number, ex: #1094

Closes #
----- No tickets closed -----

## Description of Proposed Changes
1. Upon loading the employees tab, you should be able to create a new employee
1. Has an affordance for department and computer
1. This PR does not satisfy the requirement for updating the has_been_assigned from False to True

## Steps to Test
1. Make sure virtual machine is running
1. Run migrations
1. Run server

## Outline the steps to test
1. Open db.sqlite3 in DB Browser after migrations have been created
1. Create two departments in DB Browser
1. Create two computers in DB Browser
1. Save DB Browser
1. Redirect to http://127.0.0.1:8000/WorkforceManagement/employees/
1. Click on 'Create New Employee Account'
1. Create a new employee by adding a name, start date and selecting a department and computer from the dropdown
1. Once employee has been created, verify that the employee is in the employee list
1. Create a second employee if you're just having a ton of fun and want to create another one
```
git fetch --all
git checkout js_form_employees
```



## Impacted Areas in Application

##### Added

	WorkforceManagement/forms/Employee_Form.py
	WorkforceManagement/views/Employee_View.py
	WorkforceManagement/templates/Employee_Form.py

##### Modified

	WorkforceManagement/urls.py

## Mentions @username
@MrErin @kimberly-bird @CashewRose @obscure-shadow 
Tag users that need to review this code

## Testing

There is No Testing on this PR

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
